core/driver/sources/src/main/resources/reference.conf

```diff
@@ -34,43 +46,96 @@ probe {
   // `probe.workspace.existing` can be used for scenarios where you want to reuse an existing workspace instead of
   // copying a workspace each time when tests run. It might be beneficial for big projects / repos.
+  //
   #   existing = "/Users/my_username/code/my_repository"
+
   # }
 
   // `resolvers` is for `org.virtuslab.ideprobe.config.DependenciesConfig.Resolvers` - default values below
   resolvers {
-    intellij {
-      repositories = []
-    }
-    // default configuration for `plugins` is Plugins(None) - adding possible parameters in comments for reference
-    plugins {
-      // `repository.uri` is the URI of the plugin repository - use this config if you want to use a non-standard
-      // plugin repository. The standard one used by ide-probe is: https://plugins.jetbrains.com/plugin/download
-      # repository.uri = ""
-    }
-    jbr {
-      repositories = []
-    }
+
+    // `intellij.repositories` contain patterns which should be resolved by ide-probe to download proper IntelliJ IDEA
+    // edition (Community or Ultimate) from proper intelliJ repository. Words inside square brackets
+    // are parts of the pattern that are replaced with proper values before download attempt. Current replacements
+    // are as follows:
+    // "[format]" -> value from `probe.intellij.version.format` (".zip" by default)
+    // "[revision]" -> value from `probe.intellij.version.release` - or from `probe.intellij.version.build` if `release` not found
+    // "[build]" -> value from `probe.intellij.version.build`
+    // "[version]" -> value from `probe.intellij.version.release` - or from `probe.intellij.version.build` if `release` not found
+    //
+    // If you want to use the Ultimate edition, replace "ideaIC" occurences with "ideaIU" in your config file. Community
+    // edition ("ideaIC") is the default one.
+    //
+    // You can also add your custom intellij repository on top of the 4 default repositories, so that ide-probe will
+    // first try using the IntelliJs downloaded from your repository if present. If ide-probe doesn't find the IntelliJ
+    // distribution in your repository, it will use the default 4 repositories listed below. To achieve that, specify
+    // your custom repository pattern and add the `official` or `default` keyword on another line. Example below:
+    // intellij.repositories = [
+    //   "file:///"${HOME}"/.my_intellij_idea_downloads/[revision]/ideaIC-[revision][format]",
+    //   official
+    // ]
+    //
+    // Moreover, you can add a pattern pointing to a directory of an installed IntelliJ instance instead of
+    // pattern pointing to a .zip/.dmg file. ide-probe will recognize if the pattern points to a directory with
+    // installed IntelliJ or to a .zip/.dmg file. For example:
+    // intellij.repositories = [
+    //   "file:///"${HOME}"/.pluginVerifier/ides/IC-[revision]/",
+    //   official
```

Comment:
Hmmm it's not explained what `official` does 🤔

Reply:
Ok, but I added a description in comments:
```
If ide-probe doesn't find the IntelliJ distribution in your repository, it will use the default 4 repositories listed below. To achieve that, specify your custom repository pattern and add the `official` or `default` keyword on another line. Example below:
```
Is it not obvious enough?

Reply:
Oh! okay, didn't notice... makes sense, should be pretty self-explanatory